src/crypto/aes.rs

```diff
@@ -175,6 +176,16 @@ pub fn decrypt_user_master_key(
     Ok(fixed_decrypted_master_key)
 }
 
+struct ConstantNonceGenerator {
```

Comment:
Another naming suggestion: `SingleNonceGenerator`

I think this should also have a comment saying something about how this is safe because we only use it once and we generate the nonce from a CSPRG.